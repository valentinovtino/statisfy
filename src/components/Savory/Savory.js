import React, { Component } from 'react';
import FoodHolder from '../foodHolder/foodHolder';
import { withRouter } from 'react-router';

const Savory = () => {

  return (
    <div>   
      {FoodHolder}
    </div>
  )
}

export default withRouter(Savory);