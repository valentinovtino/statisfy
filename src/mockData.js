export const mockObj = {
  "count": 30,
  "recipes": [
    {
      "publisher": "Two Peas and Their Pod",
      "f2f_url": "http://food2fork.com/view/54373",
      "title": "Focaccia Bread",
      "source_url": "http://www.twopeasandtheirpod.com/focaccia-bread/",
      "recipe_id": "54373",
      "image_url": "http://static.food2fork.com/FocacciaBread36db7.jpg",
      "social_rank": 99.99172380177085,
      "publisher_url": "http://www.twopeasandtheirpod.com"
    }
  ]
};

export const mockCleanedDataObj = 
    [{
      "publisher": "All Recipes",
      "f2f_url": "http://food2fork.com/view/12142",
      "title": "Easy Indian Butter Chicken",
      "source_url": "http://allrecipes.com/Recipe/Easy-Indian-Butter-Chicken/Detail.aspx",
      "recipe_id": "12142",
      "image_url": "http://static.food2fork.com/40062066fa.jpg",
      "social_rank": 99.99792034435711,
      "publisher_url": "http://allrecipes.com"
    }];

export const userCleanedData =
[{"id": "54373",
  "img": "http://static.food2fork.com/FocacciaBread36db7.jpg", 
  "recipe": "http://www.twopeasandtheirpod.com/focaccia-bread/", 
  "title": 
 "Focaccia Bread"}];
