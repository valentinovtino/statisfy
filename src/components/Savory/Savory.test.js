import Savory from './Savory';
import { userCleanedData } from '../../mockData';

describe('Savory', () => {
  it('should render a card', () => {
    expect(Savory).toMatchSnapshot();
  });

  it('should return clean data obj', () => {
    let 
  })

});