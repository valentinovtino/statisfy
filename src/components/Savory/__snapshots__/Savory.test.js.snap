// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Savory should render a card 1`] = `[Function]`;
